zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -4915,6 +4987,28 @@ components:
         signalKey:
           description: The unique ID of the signal that was broadcast.
           type: string
+    ResourceResult:
+      type: object
+      properties:
+        resourceName:
+          description: Resource name of this resource.
+          type: string
+        version:
+          description: Version of this resource.
+          type: integer
+          format: int32
+        versionTag:
+          description: Version tag of this resource.
```

🔧 Let's align to other endpoints:

```suggestion
          description: The version tag of the resource, as parsed during deployment.
```

---

I noticed that we use "as parsed during deployment" in some other places, but I'm not sure if it's really useful here, given that the version tag is a custom string provided by the user at development time.

---

I don't have a strong opinion on this. If you look around the rest-api doc, the usage of the term ID is anyways not documented consistently. However, I'd rather see it aligned on any of the currently used conventions rather than adding another.